Tools/ApplicationDebugger/jacobi/third-party-programs.txt

```diff
@@ -0,0 +1,253 @@
+oneAPI Code Samples - Third Party Programs File
```

Comment:
Is this file required? Please see my comment in the README.

Reply:
It is not required unless your using third party components

Reply:
Removed.